classes/hypeJunction/Scraper/Services/Linkify.php

```diff
@@ -165,7 +165,7 @@ public function callbackUsername($matches) {
 			'data-value' => $username,
 			'data-guid' => $user->guid,
 			//'data-icon' => $user->getIconURL('tiny')
-				), $user->getDisplayName());
+				), $user->name);
```

Comment:
I would prefer to check if it's callable, and if not fallback to just using name attribute.